playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP.json

```diff
@@ -0,0 +1,90 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "Enrich IP",
+    "aliasName": null,
+    "tag": null,
+    "description": "Gets triggered on enrichIP command when used in Microsoft Teams",
```

Comment:
This PB desc also needs to be modified.
Suggestion: This playbook triggers after hitting an enrichIP command using FortiSOAR for Microsoft Teams Bot

Reply:
Updated suggested description